app/concepts/todo/create.rb

```diff
@@ -1,50 +1,17 @@
 class Todo::Create < Trailblazer::Operation
 
-  # Logic for creating a TODO
```

Comment:
just wanna share this good article about comments https://robots.thoughtbot.com/letting-your-code-speak-for-itself . Good work btw. :+1: :